### ISSUE TITLE
Fonction cadastre installée mais non visible

### ISSUE THREAD

**Issue by tdeleye** [3liz/lizmap-cadastre-module#6]
Bonjour,
J'ai suivi l'installation le module à partir de : https://github.com/3liz/lizmap-cadastre-module
Description rapide de ma configuration :
**Serveur ubuntu 18.04 :** 
- lizmap-web-client-3.3.0
- qgis-server
- module cadastre sous lizmap-web-client-3.3.0/lizmap/lizmap-modules/cadastre/
- BDD cadastre sous postgreSLQ

Sur mon poste (windows)
Je crée mon projet QGS avec le module cadastre qui pointe vers la BDD postgis installée sur le serveur
Je règle les paramètres (propriétés du projet, extension lizmap).

Et je transfère le projet sur le serveur avec Filezilla

Résultat :
Le projet s'affiche correctement, mais le mode cadastre est inexistant dans la barre de menu latérale gauche.

Il doit y avoir un petit truc qui m'échappe ??

???

**Comment by Gustry** [comment]
Pouvez-vous regarder ce ticket https://github.com/3liz/lizmap-cadastre-module/issues/2 ?

**Comment by tdeleye** [comment]
Oui, j'avais même pensé poster mon post à la suite de celui-ci.
J'ai bien nommé mon répertoire "cadastre", par contre je n'arrive pas à localiser les logs de QGIS Server.
Mon compte a bien les doits sur le groupe "Cadastre_Lizmap".
De plus, j'ai un doute sur les variables du projet, les miennes ressemblent plutot à ça :
![image](https://user-images.githubusercontent.com/72381775/95068400-ebfb0d00-070d-11eb-989e-e942e75f774c.png)

**Comment by Gustry** [comment]
Oui, comme mentionné dans https://github.com/3liz/lizmap-cadastre-module/issues/2#issuecomment-608462199 vous devez utiliser le traitement "Configuration du projet" qui va ajouter les variables dans le contexte du projet. (boîte à outils Processing/Traitement)
